pkg/providers/ovirt/validation/vm-import-validation.go

```diff
@@ -106,27 +111,68 @@ type VirtualMachineImportValidator struct {
 }
 
 // NewVirtualMachineImportValidator creates ready-to-use NewVirtualMachineImportValidator
-func NewVirtualMachineImportValidator(client client.Client) VirtualMachineImportValidator {
+func NewVirtualMachineImportValidator(client client.Client, kubevirtClient kubecli.KubevirtClient) VirtualMachineImportValidator {
 	return VirtualMachineImportValidator{
-		Validator: &validators.ValidatorWrapper{},
+		Validator: validators.NewValidatorWrapper(kubevirtClient),
 		client:    client,
 	}
 }
 
 // Validate validates whether VM described in VirtualMachineImport can be imported
-func (validator *VirtualMachineImportValidator) Validate(vm *ovirtsdk.Vm, vmiCrName *types.NamespacedName) error {
+func (validator *VirtualMachineImportValidator) Validate(vm *ovirtsdk.Vm, vmiCrName *types.NamespacedName, mappings *v2vv1alpha1.OvirtMappings) error {
```

Comment:
I do not see a place where we merge partial mapping from VM import object with global `OvirtMappings`. VM import specific can override global mapping we should merge them before validating.

Reply:
That's right - the code hasn't been plugged into reconciliation loop yet. The mapping will be merged there, before being passed to validation.

Reply:
exactly. the input for this method will be the merged mappings where the embedded mappings overrides the master mappings.